ddtrace/contrib/tornado/futures.py

```diff
@@ -0,0 +1,33 @@
+from ddtrace import tracer
+from ddtrace.context import Context
+
+
+def _wrap_submit(func, instance, args, kwargs):
+    """
+    Wrap `Executor` method used to submit a work executed in another
+    thread. This wrapper ensures that a new `Context` is created and
+    properly propagated using an intermediate function.
+    """
+    # create a new Context with the right active Span
+    # TODO: the current implementation doesn't provide the GlobalTracer
+    # singleton, so we should rely in our top-level import
+    ctx = Context()
+    current_ctx = tracer.get_call_context()
+    ctx._current_span = current_ctx._current_span
```

Comment:
With this pending PR https://github.com/DataDog/dd-trace-py/pull/325 will we have to copy other attributes?

Reply:
Absolutely yes, but I prefer to ship that change + release 0.9.3 and then handle per framework migration to the new API.